data/OR/incentives.json

```diff
@@ -0,0 +1,892 @@
+  {
+    "id": "OR-28",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "or_energyTrustOfOregon_savingsWithinReach",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1.5,
+      "unit": "square_foot"
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$1.50 per sq ft of attic insulation. Income-qualified customers. Existing insulation must be R-18 or less. Must insulate to R-38 or greater."
+    },
+    "low_income": "or-energy-trust-of-oregon"
+  },
+  {
+    "id": "OR-29",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1.25,
+      "unit": "square_foot"
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$1.25 per sq ft of attic insulation for manufactured homes. Existing insulation must be R-18 or less. Must insulate to R-30 or greater."
+    }
+  },
+  {
+    "id": "OR-32",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$500 discount on a ductless heat pump with additional heating source. The heat pump must replace zonal electric as the primary heat source."
+    },
+    "start_date": "2023"
+  },
+  {
+    "id": "OR-33",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "other",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$250 discount on qualifying heat pump controls. Must lock electric auxiliary heat out @ 35F or lower and have an outdoor temp sensor / be web-enabled."
+    }
+  },
+  {
+    "id": "OR-34",
+    "authority_type": "utility",
+    "authority": "or-pacific-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "electric_vehicle_charger",
+    "program": "or_pacificPower_pacificPowerRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $500 rebate for qualifying hardwired level 2 chargers, capped at 75% of total costs. Must be installed by a licensed electrician."
```

Comment:
For OR-34 through OR-37, these are better represented as amount type percent (with amount numbers as 0.75), and the current dollar values should become amount maximums. The descriptions don't need to change.